nexus/src/app/instance.rs

```diff
@@ -1664,7 +1664,7 @@ impl super::Nexus {
                         vmm.runtime.state,
                     )))
                 }
-                DbVmmState::Destroyed => Err(Error::invalid_request(
+                DbVmmState::Destroyed | DbVmmState::SagaUnwound => Err(Error::invalid_request(
                     "cannot connect to serial console of destroyed instance",
```

Comment:
I think we should tweak this so that the error message says the instance is Stopped in this case. (It's not Destroyed from the user's perspective since they were able to refer to it to ask for its serial console.)

The message here isn't new, of course; I think the difference is that today, when a VMM dies or a start saga gets unwound, the instance's active Propolis ID is cleared before the VMM actually gets destroyed. When the start saga starts using this state, it's expected that an instance will have a non-NULL Propolis ID that points to a VMM in this state, so this match arm will be much more reachable than before.

Reply:
yup, makes sense. will update it!

Reply:
hmm, perhaps we should rephrase this to match the wording of the error message in the previous match arm, so that we don't have to different messages that both suggest that the instance is "stopped", but with different wording (which implies to the user that they are Secretly Different...)